app/assets/stylesheets/administrate/components/_buttons.scss

```diff
@@ -34,3 +34,8 @@
     }
   }
 }
+
+.scope-buttons {
```

Comment:
DISCLAIMER: I'm mostly a backend developer and I've never used Sass before :( But I like it!!!

Totally agree Tyson, I put it here because in that moment the project still didn't have the components/_buttons.scss file, and i thought base/buttons.scss was the best place to put that style.

In that moment the style just was "margin: 3px", value and unit that i used to have the same as the $base-border-radius variable. Then came this merge with your (awesome;) work:

https://github.com/nando/administrate/commit/c026d414882

...and i had to put the "padding-left: 1.5em" style in order to separate them from the border using the value that the browser told me the header was using...